### ISSUE TITLE
Prototype interfacing the interpreter with an external package manager

### ISSUE THREAD

**Issue by yannham** [tweag/nickel#1645]
As part of #1585, and after #1643 and #1644 , the next step is to write a proof of concept interfacing Nickel with one external package manager (not necessarily the default one, it can be the easiest to interface with) in order to import files from an external source.

**Comment by yannham** [comment]
This has been superseded by #1983 (RFC006). In particular it was decided to go with a bespoke Nickel package manager in the end.
